pages/nfc/_id.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <main>
+    <div>Redirecting</div>
+    <div>
+      {{ docSnap }}
+    </div>
+  </main>
+</template>
+
+<script>
+import { db } from '~/plugins/firebase'
+
+export default {
+  async asyncData() {
+    const pageId = $nuxt.$route.params.id
+    const docRef = db.collection('profiles').doc(pageId)
```

Comment:
You should find document in collection nfc-cards, please see acceptance criteria